modules/data/pages/contracts-and-contexts.adoc

```diff
@@ -123,7 +123,7 @@ In `Edit` mode, additional `persistenceId_string` input are generated to ensure
 In `Edit` mode, you have the possibility to generate read only widgets for attributes related to the contract but not in the contract. +
 The following example describes the logic:
 
-image::images/formGenerationReadOnly.svg[Read only example]
+image:images/formGenerationReadOnly.svg[Read only example]
```

Comment:
Fixing this because the image did not show anymore, since conversion from MD to Asciidoc 😞